### PR TITLE
Adding performance logging around TestBasicOps.

### PR DESCRIPTION
**Pull request by sduskis**

### CODE REVIEW

File: bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
 public abstract class AbstractTest {
   protected DataGenerationHelper dataHelper = new DataGenerationHelper();
 
+  protected Logger logger = new Logger(this.getClass());
```
Comment (kevinsi4508) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java]: static?

Comment (sduskis) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java]: We can't use "this" with static.  "this" is needed to get a logger that will print out the classname.

File: bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBasicOps.java [hunk 2]
```diff
@@ -124,12 +126,28 @@ public void testNullQualifier() throws IOException {
...
+    testBigValue(false);
+  }
+
+  private void testBigValue(boolean doGet) throws IOException {
     // Initialize variables
     byte[] testRowKey = dataHelper.randomData("testrow-");
     byte[] testQualifier = dataHelper.randomData("testQualifier-");
     byte[] testValue = new byte[(10 << 20) - 1024];  // 10 MB - 1kB
     new Random().nextBytes(testValue);
+    doTest(doGet, testRowKey, testQualifier, testValue);
```
Comment (kevinsi4508) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBasicOps.java]: if doTest() is doing the testing what is putGetDeleteExists() after doTest()?

File: bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBasicOps.java [hunk 3]
```diff
@@ -163,29 +181,54 @@ public void testPutAlmostTooBigValue() throws IOException {
...
     // Get
     Get get = new Get(rowKey);
     get.addColumn(COLUMN_FAMILY, testQualifier);
-    Result result = table.get(get);
-    Assert.assertTrue(result.containsColumn(COLUMN_FAMILY, testQualifier));
-    List<Cell> cells = result.getColumnCells(COLUMN_FAMILY, testQualifier);
-    Assert.assertEquals(1, cells.size());
-    Assert.assertArrayEquals(testValue, CellUtil.cloneValue(cells.get(0)));
 
+    // Do the got on some tests, but not others.  The rationale for that is to do performance
```
Comment (kevinsi4508) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestBasicOps.java]: /s/got/get/

**Comment by AngusDavis** [comment]
Does it make sense to plug into CallTiming (if they were brought into the github project, not certain) interceptors in some way? We can are about both the processing / queueing delay as well as individual call times, yes?

File: bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java (hunk 1 above)
Comment (kevinsi4508) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java]: not a big deal, use can use AbstractTest.class if you like.

**Comment by sduskis** [comment]
It does make sense to add on more information about performance.  I'm not sure what CallTiming is, but if it helps, I'm all for it.  

With this PR, I'm hoping for a quick, intuitive addition to the existing TestBasicOps testing that will help us time the performance of Big Put and Get.

**Comment by kevinsi4508** [comment]
LGTM on my end.

File: bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java (hunk 1 above)
Comment (sduskis) [inline bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/AbstractTest.java]: When looking at a log, I feel that it's important to be able to be able to quickly tell what class produced a log message.

**Comment by AngusDavis** [comment]
RE: CallTiming, I see it didn't survive. I'll send you a link. Overall timing is a nice to have, but doesn't really tell us a whole lot (did the OAuth servers flake on us, did we have a token refresh, etc).

**Comment by sduskis** [comment]
I'm pushing this so we can see the test run.  Let's talk about CallTiming and other options offline.

**Pull request merged**